### PR TITLE
[Docs] Explain how to enable plugins in Xcode and Xcode Cloud

### PR DESCRIPTION
**Pull request by czechboy0**
### Motivation

By default, adopters hit this error both in Xcode and Xcode Cloud and might not be familiar with how to resolve it.

### Modifications

Expand the FAQ to cover how to enable plugins in Xcode and Xcode Cloud.

### Result

Better docs, adopters can find a solution easier.

### Test Plan

N/A

### CODE REVIEW

**Review by simonjbeaumont — APPROVED** [review]
Thanks for rolling this up into the FAQ. We had this answer floating in a few places on various forums, so it's good to catch it here.

File: Sources/swift-openapi-generator/Documentation.docc/Articles/Frequently-asked-questions.md [hunk 1]
```diff
@@ -74,13 +74,13 @@ For details, check out <doc:Configuring-the-generator>.
 
 ### How do I enable the build plugin in Xcode and Xcode Cloud?
 
-By default, build plugins must be explicitly enabled by the adopter.
+By default, you must explicitly enable build plugins before they are allowed to run.
 
-In Xcode, the first time you add a build plugin as a dependency of your project or package, the build fails and requires you to enable the plugin.
+Before a plugin is enabled, you might encounter a build error with the message _"OpenAPIGenerator" is disabled_.
```
Comment (simonjbeaumont) [inline Sources/swift-openapi-generator/Documentation.docc/Articles/Frequently-asked-questions.md:79]:
```suggestion 
Before a plugin is enabled, you will encounter a build error with the message `"OpenAPIGenerator" is disabled`.
```

**Review by simonjbeaumont — APPROVED** [review]
Looks good.

**Comment by theoriginalbit** [comment]
It's potentially worth adding a comment to the script to point out that the misspelling of validation in the plist key is "correct" as Xcode Cloud/Xcode expects it.

I've had many devs before try to "fix" the typo and then wonder why it doesn't work.

**Comment by simonjbeaumont** [comment]
> It's potentially worth adding a comment to the script to point out that the misspelling of validation in the plist key is "correct" as Xcode Cloud/Xcode expects it.
> 
> I've had many devs before try to "fix" the typo and then wonder why it doesn't work.

Ha! Nice spot. I'm slightly embarrassed that, while I've used this many times, I never even noticed :)

Yes, if you open a PR, we'd take it.

**Comment by theoriginalbit** [comment]
> Ha! Nice spot. I'm slightly embarrassed that, while I've used this many times, I never even noticed :)

I'd like to take credit for noticing it, but all the credit has to go to the spell checker. :D I have it turned on in Xcode, so it points out spelling mistakes to me all the time.

**Pull request merged**